pages/glossary/implicit-role.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Implicit Semantic Role
+key: implicit-role
+---
+
+The Implicit Semantic Role is the default [semantic role](#semantic-role) of each element that is used when no valid [explicit semantic role](#explicit-role) is specified. Elements with no [role attribute](https://www.w3.org/TR/html/dom.html#aria-role-attribute), or with a [role attribute](https://www.w3.org/TR/html/dom.html#aria-role-attribute) containing no valid token, are mapped to their implicit role.

```

Comment:
I'm not sure about the use of "default" in the definition. I would prefer to remove it from the sentence.